src/archanaNJun20/RemoveDuplicateNames.java

```diff
@@ -0,0 +1,22 @@
+package archanaNJun20;
+
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class RemoveDuplicateNames {
+	void removeDuplicateNamesFromString(String[] input) {
+		Set<String> nameString = new LinkedHashSet<>();
+		for(String name:input) 
+			nameString.add(name);
```

Comment:
You can directly write like this instead of using loop
"Set<String> inputSet = new LinkedHashSet<String>(Arrays.asList(input));"